HelpAssistHer/help-assist-her
# Update mini app header

## Description
Style the header for phone and desktop designs. 

Note: it's not perfect, there is still a weird state at the tablet size because of the size of the picture. It also might be too big at the desktop level. I'm going to continue to play around with it, but I think it will be easier to see how it all looks once it's merged in with all the other changes I've made. 

Also note: the copy in the designs is not the official wording, so it's not going to match.

## Test Plan
1. Go to http://localhost:4000/mini-app
1. See this, and then play around with resizing the browser and see how it changes 
<img width="1437" alt="screen shot 2018-11-01 at 9 19 36 am" src="https://user-images.githubusercontent.com/6378435/47864629-56cf9b00-ddb7-11e8-9381-716a090043f7.png">
